pom.xml

```diff
@@ -120,6 +120,22 @@
             <artifactId>jackson-core</artifactId>
             <version>${jackson.version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.projectlombok</groupId>
```

Comment:
Wondering if we should document somewhere how Lombok annotations will "generate" some code for us. Mostly to help contributors not familiar with the library understand where LOGGER is getting instantiated. Perhaps a small note in the README? Maybe overkill if we don't plan to introduce lombok further into the codebase.

Reply:
done.